alpha/lib/model/LiveEntry.php

```diff
@@ -1210,4 +1211,9 @@ public function getRecordedEntryConversionProfile()
         }
 	    return $conversionProfileId;
     }
+
+	public function isLowLatencyEntry() : bool
```

Comment:
having return declaration type wont work on low php versions

Reply:
Move return type to comment